sample/src/main/java/com/uber/rave/sample/DefaultSerializer.java

```diff
@@ -0,0 +1,53 @@
+package com.uber.rave.sample;
+
+import com.uber.rave.sample.github.model.Owner;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+
+/**
+ * This uses the default Java serializer.
+ */
+class DefaultSerializer {
+
+    static byte[] serialize(Owner owner) {
```

Comment:
Consider just using `gson` (which we already pulled in) to serialize things here and writing them to disk with `SharedPreferences`. This isn't writing to disk at the moment.